alkira/resource_alkira_service_zscaler.go

```diff
@@ -0,0 +1,242 @@
+package alkira
+
+import (
+	"log"
+
+	"github.com/alkiranet/alkira-client-go/alkira"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func resourceAlkiraServiceZscaler() *schema.Resource {
+	return &schema.Resource{
+		Description: "Manage Zscaler firewall.",
+		Create:      resourceZscaler,
+		Read:        resourceZscalerRead,
+		Update:      resourceZscalerUpdate,
+		Delete:      resourceZscalerDelete,
+
+		Schema: map[string]*schema.Schema{
+			"alkira_internet_connector_id": {
```

Comment:
I see... let's be consistent, I think it could be simply `connector_internet_exit_id`, to be consistent.